.github/workflows/luis_pr.yaml

```diff
@@ -69,7 +69,7 @@ jobs:
         status=$(echo "$response" | jq '.Status' | xargs)
         if [ "$status" == "Success" ]
         then
-          appId=$(echo "$response" | jq '.id' | xargs)
+          appId=$(echo "$response" | jq '.id' | tr -d '\\"')
```

Comment:
what's happening here?

Reply:
@AmirGeorge for someone reason in the version there are escaped double quotes in the appId string so we're removing them using tr